packages/date-picker/src/_primitives/Calendar/Calendar.scss

```
@@ -98,7 +98,8 @@ $rdp-cell-size: 40px;
     border: $border-solid-border-width $border-solid-border-style transparent;
     border-radius: $cell-border-radius;
 
-    &:hover {
+    &:hover,
+    &:global(.story_hover) {
```

Since it's a global class we should be more specific ([bem is helpful here](http://getbem.com/naming/)), otherwise we might end up with global styles from different component sticker sheets affecting others.

```suggestion
    &:global(.story__datepicker__calendar--hover) {
```